### DISCUSSION TITLE
Use chunk labels as default folded code block titles

### DISCUSSION THREAD

**Discussion by daaronr** [quarto-dev/quarto-cli#2489] [Feature Requests]
Folded code is great but I like to separate chunks by function. Then in one section I'll get a lot of folded blocks that are just labelled "code" and I can't remember which one to unfold. It would be super helpful if the title of each code block (the unfolded part) was set to the chunk label. Ok I realise you can set these individually. But maybe a default to chunk label would be good.

**Comment by mcanouil** [comment]
Thanks for the suggestion!

To note, `code-summary` can be changed manually as explained, see <https://quarto.org/docs/reference/formats/html.html#code>.
